src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall().toString());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall().toString());
```

Comment:
1. It is not needed to create 3 instances of the Lottery class here. We can create one instance and call getRandomBall() 3 times
2. Lottery instances and not instances of the Ball class, to be named ball1, ball2, ball3
3. We need to override the toString() method of the [Ball](https://github.com/TarasDovganyuk/lottery-test-task-review/pull/1/files#:~:text=public%20class-,Ball,-%7B) class, if we don't want to see just the reference, printed to console